### PR TITLE
Fixed logic to check partial withdrawls haven't been proceeded by vot…

### PR DESCRIPTION
**Pull request by moose-code**
…ing. Removed condition that you must not have voted to top up

### CODE REVIEW

File: contracts/contracts/PoolDeposits.sol [hunk 1]
```diff
@@ -215,11 +231,24 @@ contract PoolDeposits {
     }
   }
 
+  /*
```
Comment (JasoonS) [inline contracts/contracts/PoolDeposits.sol:234]: I think we can remove this, we basically just need to go through the modifiers and check if  `userHasNoProposal`.

File: contracts/test/userAdditionalDeposit.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      amount: mintAmount,
+    });
+
+    let deposit = await poolDeposits.depositedDai.call(accounts[1]);
+
+    await time.increase(time.duration.seconds(8640001)); // 100 days and 1 second - required to `voteEmergency`
+
+    await poolDeposits.voteEmergency({ from: accounts[1] });
+
+    // check that the partial withdraw fails.
```
Comment (moose-code) [inline contracts/test/userAdditionalDeposit.js:86]: check the partial deposit fails

File: contracts/test/userAdditionalDeposit.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    await expectRevert(
+      poolDeposits.deposit(mintAmount, { from: accounts[1] }),
+      'User does not have enough DAI'
+    );
+
+    // User has joined the pool - no changes
+    assert.equal(mintAmount, deposit.toString());
+  });
+
+  // Q: do we need to implement another test checking you cannot add additional deposit once state of emergency declared?
```
Comment (moose-code) [inline contracts/test/userAdditionalDeposit.js:186]: Nope that is handled already in the emergency tests I'm pretty sure :)

Comment (moose-code) [reply contracts/test/userAdditionalDeposit.js:186]: I don't see tests checking dao logic based on added amounts (they can no longer vote that iteration etc...)

**Pull request merged**